docs/node-observ-lib/README.md

```diff
@@ -0,0 +1,111 @@
+# Node exporter observ lib
+
+This jsonnet observ lib can be used to generate observ package for node exporter.
+
+## Import
+
+```sh
+jb init
+jb install https://github.com/grafana/node_exporter/docs/node-observ-lib
+```
+
+## Examples
+
+### Example 1: Basic example
+
+You can use observ-lib to fill in monitoring-mixin structure:
+
+```jsonnet
+// mixin.libsonnet file
+local nodelib = import 'node-observ-lib/main.libsonnet';
+
+local linux =
+  nodelib.new()
+  + nodelib.withConfigMixin({
```

Comment:
Maybe we should add a little explanation/example with separate configuration?

Reply:
added